docs/docs/features/underglow.md

```diff
@@ -162,3 +162,15 @@ CONFIG_ZMK_RGB_UNDERGLOW=y
 # Use the STRIP config specific to the LEDs you're using
 CONFIG_WS2812_STRIP=y
 ```
+
+## Change `chain-length` Only
+
+A common need is to change only the `chain-length` value for a `led_strip` node that is otherwise fully configured in the board definition, often because the default value accounts only for either the downward facing LEDs or the upward facing LEDs under each key. This can be done by [changing that property](https://zmk.dev/docs/config#changing-devicetree-properties) in the `*.keymap` file by adding a stanza like this one outside of any other node (i.e. above or below the `/` node):
```

Comment:
```suggestion
A common need is to change only the value of the `chain-length` property for a `led_strip` node that is otherwise fully configured in the board definition, often because the default value accounts only for either the downward facing LEDs or the upward facing LEDs under each key. This can be done by [changing that property](../config/index.md#changing-devicetree-properties) in the `<keyboard>.keymap` file by adding a stanza like this one outside of any other node (i.e. above or below the `/` node):
```
(We standardized on `.md` links, but please double check the link works after the change as I don't have a dev environment right now)

Reply:
Your suggestions for changing the name and location of the section prompted me to reconsider the language of the body as well. It seems like it ought to lead either with the objective we're attempting to reach or the symptoms of the problem we're trying to solve, rather than a summary of the solution. How about this:

## Modifying the number of LEDs

A common issue when enabling underglow is that some of the installed LEDs do not illuminate. This can happen when a board's default underglow configuation accounts only for either the downward facing LEDs or the upward facing LEDs under each key. On a split keyboard, a good sign that this may be the problem is that the unilluminated LEDs on each half are symmetrical.

The number of underglow LEDs is controlled by the `chain-length` property in the `led_strip` node. You can [change the value of this property](../config/index.md#changing-devicetree-properties) in the `<keyboard>.keymap` file by adding a stanza like this one outside of any other node (i.e. above or below the `/` node):
...

Reply:
Sure, looks good to me!